learning/prompts/documentation-lookup/04_basic_pcollections.md

```diff
@@ -0,0 +1,22 @@
+Prompt:
+What is a PCollection in Apache Beam?
+Response:
+A `PCollection` in Apache Beam is a core abstraction representing a distributed, multi-element data set or data stream. It’s the primary data structure used in Apache Beam pipelines to handle large-scale data processing in batch and streaming modes.
```

Comment:
This is a bit different than the definition we have in the documentation:

https://beam.apache.org/documentation/basics/#pcollection

Not sure if one is better than the other, or if both are correct.

Reply:
Updated definition